src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.DataConverter;
+import core.basesyntax.dao.FileReader;
+import core.basesyntax.dao.FileWriter;
+import core.basesyntax.dao.impl.DataConverterImpl;
+import core.basesyntax.dao.impl.FileReaderImpl;
+import core.basesyntax.dao.impl.FileWriterImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitShop;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.impl.FruitShopImpl;
+import core.basesyntax.service.impl.OperationStrategyImpl;
+import core.basesyntax.service.impl.ReportGeneratorImpl;
+import core.basesyntax.service.operations.BalanceOperation;
+import core.basesyntax.service.operations.OperationHandler;
+import core.basesyntax.service.operations.PurchaseOperation;
+import core.basesyntax.service.operations.ReturnOperation;
+import core.basesyntax.service.operations.SupplyOperation;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] arg) {
+        FileReader fileReader = new FileReaderImpl();
+        List<String> inputReport = fileReader.read("reportToRead.csv");
```

Comment:
Avoid hardcoding file paths. It's better to define file paths as constants in the Main class or pass them as command-line arguments. This will make your code more flexible and easier to update or configure.